docs/spelling_wordlist.txt

```diff
@@ -18,6 +18,7 @@ dataset
 datasets
 Debian
 Deis
+dev
```

Comment:
As per http://sphinxcontrib-spelling.readthedocs.org/en/latest/customize.html this should be "a list of words known to be spelled correctly but that do not appear in the language dictionary". Of course, we don't have to stick to that (and you can see in this file that we haven't always) but I prefer to wrap the offending "libssl-dev" in double backquotes, as it is kinda-sorta a code sample.